src/views/pages/app/authenticate/Authenticate.tsx

```diff
@@ -15,332 +21,417 @@ import { config, Env } from "src/ultils";
 import { ApiErrorModal } from "../../../../components/common/ApiErrorModal";
 
 export enum AuthenticateType {
```

Comment:
Please do not change the code format. 

And please run `lint:fix` before submitting